src/gui/dialogs/editor/edit_unit.hpp

```diff
@@ -58,6 +58,8 @@ class editor_edit_unit : public modal_dialog
 
 	std::vector<config> align_list_, race_list_, movetype_list_, defense_list_, resistances_list_, usage_type_list_;
 	std::vector<config> specials_list_, abilities_list_;
+	// hardcoded list of untranslatated alignment strings
+	std::vector<std::string> align_list_untrans_ = {"lawful", "chaotic", "neutral", "liminal"};
```

Comment:
Why can't you use `unit_alignments::values`?

Reply:
Didn't know about that, updated.